kyuubi-common/src/main/scala/org/apache/kyuubi/session/SessionManager.scala

```diff
@@ -69,6 +69,12 @@ abstract class SessionManager(name: String) extends CompositeService(name) {
   private val timeoutChecker =
     ThreadUtils.newDaemonSingleThreadScheduledExecutor(s"$name-timeout-checker")
 
+  @volatile private var _limiter: Option[SessionLimiter] = None
```

Comment:
we do not expose this value, so `limiter` is enough

Reply:
> we do not expose this value, so `limiter` is enough

thanks, i have fixed it